docs/guide/advanced/navigation-guards.md

```diff
@@ -39,7 +39,7 @@ Every guard function receives three arguments:
 ```js
 // BAD
 router.beforeEach((to, from, next) => {
-  if (!isAuthenticated) next('/login')
+  if (to.path !== '/login' && !isAuthenticated) next('/login')
```

Comment:
```suggestion
  if (to.name !== 'Login' && !isAuthenticated) next('/login')
```
let's use a named route instead for both examples

Reply:
If we're doing that, would it be better to use it in the call to next as well for consistency?

`next({ name: 'Login' });`

I'm actually using the name in my code, but I didn't want to mix concepts and the path should be fine because it doesn't contain query params and such.

Reply:
Yes, that makes sense!